tests/python/unittest/test_tvmscript_spans.py

```diff
@@ -18,26 +18,26 @@
 import inspect
 
 import tvm
-import tvm.script
-from tvm import tir
+from tvm.script import tir as T
 
 
+@T.prim_func
 def loops() -> None:
-    for i in tir.parallel(0, 2):
-        for j in tir.serial(0, 1):
-            for z in tir.vectorized(3, 4):
-                tir.evaluate(0)
+    for i in T.parallel(0, 2):
+        for j in T.serial(0, 1):
+            for z in T.vectorized(3, 4):
+                T.evaluate(0)
 
 
 def test_loops():
-    _, start_line = inspect.getsourcelines(loops)
-    parsed = tvm.script.tir(loops)
+    start_line = 25
```

Comment:
Using relative line numbers in the tests make it easier to reorganize the tests. Is there a reason that is not possible with any more?

Reply:
Yes, I 100% agree that relative line numbers are better.

However, we ensure that functions need have decorators so that we can not get the Python function Object ti get line number.